DigitalLearningSolutions.Web/Controllers/TrackingSystem/Delegates/EditDelegateController.cs

```diff
@@ -41,9 +41,9 @@ PromptsService registrationPromptsService
         public IActionResult Index(int delegateId)
         {
             var centreId = User.GetCentreId();
-            var delegateUser = userService.GetUsersById(null, delegateId).delegateUser;
+            var delegateUser = userService.GetDelegateById(delegateId);

```

Comment:
`delegate` is a keyword in C# so I'm not able to name a variable simply "delegate". I thought delegateUser was the best alternative, unless you have a better one in mind?

Reply:
Ah, I didn't realise that. How very annoying. I think delegateUser is not appropriate because there is a DelegateUser class, and these variables are not instances of it. I think, following the naming conventions found elsewhere, maybe `Delegate` should be renamed to `DelegateEntity` (like `UserEntity`), and the instances renamed to `delegateEntity`.

Reply:
Oh yeah, forgot about that. 🤦 
Think `GetDelegateEntityById` is fine.